Core app with material ui (#1637)
SAP/luigi: core/examples/luigi-core-material-ui/README.md

# Sample Luigi CORE application written in React with Material-UI

## Overview

Example on how to create a Luigi Core based interface from scratch by disabling all built-in graphical interfaces.

> > Read the [Write a Luigi Core interface from scratch](https://docs.luigi-project.io/docs/advanced-scenarios) in our Expert Scenarios to get a detailed introduction about this implementation.

Reviewer: ```suggestion
> Read the [Write a Luigi Core interface from scratch](https://docs.luigi-project.io/docs/advanced-scenarios) section in our Expert Scenarios to get a detailed introduction on this implementation.
```

> Be aware this is only a MVP with navigation and a simple profile, currently also only a limited amount of events are available publicly from Core. Please get in touch with us, if you want to build a Core app from scratch. We are happy to get to know what you are creating and give support by exposing required events.
> 

Reviewer: ```suggestion
Be aware this is only a MVP with navigation and a simple profile, and only a limited amount of events are currently available publicly from Core. Please get in touch with us on [Slack](slack.luigi-project.io) if you want to build a Core app from scratch. We are happy to get to know what you are creating and support you with exposing required events.

```
